openshift/Makefile

```diff
@@ -15,7 +15,7 @@ verify: ## Run downstream-specific verify
 
 .PHONY: manifests
 manifests: $(KUSTOMIZE) $(YQ)
-	$(DIR)/generate-manifests.sh
+	$(DIR)/operator-controller/generate-manifests.sh
```

Comment:
So instead of /operator-framework-operator-controller/blob/main/openshift/generate-manifests.sh we are targeting /operator-framework-operator-controller/blob/main/openshift/operator-controller/generate-manifests.sh